src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Nullable.cs

```diff
@@ -14,6 +14,18 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public class AttributeTests_Nullable : CSharpTestBase
     {
+        private const string NonNullTypesAttributesDefinition = @"
```

Comment:
>NonNullTypesAttributesDefinition [](start = 29, length = 32)

We should consider making this available on CSharpTestBase, it's likely that many more tests will need the attribute.

Reply:
Added a `// PROTOTYPE` comment for now.

---
In reply to: [207043241](https://github.com/dotnet/roslyn/pull/28959#discussion_r207043241) [](ancestors = 207043241)